src/Worker/ShopwareConsoleCaller.php

```diff
@@ -57,18 +57,19 @@ public function call($command, $parameters = [])
         ];
 
         $process = \proc_open($fullCommand, $stdDescriptors, $pipes, $this->workingDirectory, null);
+        if (false === $process || false === is_resource($process)) {
+            $this->error = printf('Could not start command "%s" correctly. No valid process resource was returned', $command);
+            return false;
+        }
+
+        stream_set_blocking($pipes[0], 0);
```

Comment:
For better readability:
```suggestion
        stream_set_blocking($pipes[0], false);
```
?
The second parameter is a bool: http://php.net/manual/de/function.stream-set-blocking.php